src/javascript/core/send.js

```diff
@@ -99,6 +99,19 @@ function sendRequest(request, callback) {
 	if (!(settings.get('developer') && settings.get('no_send'))) {
 		transport.send(url, stringifiedData);
 	}
+
+	// Detect IE11
+	const ie11 = !!navigator.userAgent.match(/Trident\/7\./);
+	if (ie11 && request.category === 'page' && request.action === 'view') {
```

Comment:
Isn't there a danger of other systems that hang off spoor getting double page views?

Reply:
No, I'm rewriting the category below before resending using the image transport